tests/modules/sharedaddy/test_class.recaptcha.php

```diff
@@ -0,0 +1,55 @@
+<?php
+require dirname( __FILE__ ) . '/../../../modules/sharedaddy/recaptcha.php';
+
+class WP_Test_Jetpack_ReCaptcha extends WP_UnitTestCase {
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->site_key   = defined( 'RECAPTCHA_PUBLIC_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'sitekey';
+		$this->secret_key = defined( 'RECAPTCHA_PRIVATE_KEY' ) ? RECAPTCHA_PUBLIC_KEY : 'secretkey';
+		$this->recaptcha  = new Jetpack_ReCaptcha( $this->site_key, $this->secret_key );
+	}
+
+	public function test_get_default_config() {
+		$default_config = $this->recaptcha->get_default_config();
+		$this->assertNotEmpty( $default_config );
+		$this->assertArrayHasKey( 'language', $default_config );
+		$this->assertArrayHasKey( 'tag_class', $default_config );
+		$this->assertArrayHasKey( 'tag_attributes', $default_config );
+		$this->assertNotEmpty( $default_config['tag_attributes'] );
+		$this->assertArrayHasKey( 'theme', $default_config['tag_attributes'] );
+		$this->assertArrayHasKey( 'type', $default_config['tag_attributes'] );
+		$this->assertArrayHasKey( 'tabindex', $default_config['tag_attributes'] );
+	}
+
+	public function test_verify() {
+		// Empty response returns WP_Error.
+		$result = $this->recaptcha->verify( '', '127.0.0.1' );
+		$this->assertInstanceOf( 'WP_Error', $result );
+
+		// TODO: test succeed and failed respones. Make sure to mock wp_remote_post.
```

Comment:
Can we actually implement this `TODO`, since it's probably the most useful set of tests to have?